samples/HelloWorld/README.md

```diff
@@ -14,7 +14,7 @@ Open a new shell/command prompt window and run the following commands.
 This will create a simple Hello World console app in `Program.cs` and associated project files.
 
 ## Add CoreRT to your project
-Using CoreRT to compile your application is done via the ILCompiler NuGet package, which is [published to MyGet with the CoreRT daily builds](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.DotNet.ILCompiler).
+Using CoreRT to compile your application is done via the ILCompiler NuGet package, which is published to an Azure blob with the CoreRT daily builds.
```

Comment:
Nit: “Azure blob” sounds is too much detail - do folks reading this readme need to know this detail? I would just say “Azure” here; or do not say where the NuGet package is published to at all.